src/state/store.ts

```diff
@@ -0,0 +1,25 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
```

Comment:
this is effectively boilerplate to get stuff to run. Additional slices that you want to add to Redux should be added in rootReducer. You can also add middleware, which might be very useful for your history log.